packages/components/select/src/useSelect.ts

```diff
@@ -569,6 +571,7 @@ export const useSelect = (props, states: States, ctx) => {
     ctx.emit(UPDATE_MODEL_EVENT, value)
     emitChange(value)
     states.visible = false
+    states.hoverIndex = -1
```

Comment:
Why set hoverIndex to -1 when delete selected

Reply:
I think we should set the value of hoverIndex like virtual select

Reply:
Now the hoverIndex does not change when mouseover

Reply:
> Why set hoverIndex to -1 when delete selected?

`hoverIndex`  indicates the subscript of the option of the hover state. Therefore, when the hover state is cleared, its value is set to -1.

Reply:
> Now the hoverIndex does not change when mouseover  

This does not seem to be caused by this change. The current performance of the official website is the same. The selected options have no hover effect.